src/adv.py

```diff
@@ -37,15 +40,45 @@
 # Main
 #
 
-# Make a new player object that is currently in the 'outside' room.
+# New player object that is currently in the 'outside' room.
+playerOne = Player("Kimberlee", 'outside')
 
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+# Setting up Boolean for status of game
+active_session = True
+
+# Write a loop ('REPL parser' from the instructions) that, while the session is active:
+while active_session: 
+   
+   # Prints the current room name and description
+   # Room for improvement: print these as string literals 
+   print("You are in the...")
+   print(room[playerOne.current_room].name)
+   print("...")
+   print(room[playerOne.current_room].description)
+
+   # * Waits for user input and decides what to do.
+   user_direction = input("Enter a cardinal direction the player should move to \n n for North \n e for East \n s for South \n w for West \n q to Quit \n Your input: ").lower()
```

Comment:
nice, handling the case of user inputting uppercase command. also mentioned in lecture today: you can add `strip()` in case the user inputs extra spaces.